src/octree/mod.rs

```diff
@@ -130,7 +131,7 @@ fn relative_size_on_screen(bounding_cube: &Cube, matrix: &Matrix4<f64>) -> f64 {
         Point3::new(max.x, min.y, max.z),
         Point3::new(min.x, max.y, max.z),
     ] {
-        rv = rv.grow(clip_point_to_hemicube(&project(matrix, p)));
+        rv.grow(clip_point_to_hemicube(&project(matrix, p)));
     }
     (rv.max().x - rv.min().x) * (rv.max().y - rv.min().y)
```

Comment:
how about using `Aabb::diag` here?
```suggestion
    let diag = rv.diag();
    diag.x * diag.y
```

Reply:
Done.